code/__HELPERS/unsorted.dm

```diff
@@ -1358,6 +1358,11 @@ GLOBAL_DATUM_INIT(dview_mob, /mob/dview, new)
 	see_in_dark = 1e6
 	anchored = TRUE
 	var/ready_to_die = FALSE
+	stat = DEAD // Prevents this mob from barging into the living mobs list
+
+/mob/dview/Initialize()
```

Comment:
I swear we were supposed to override dview/New to prevent it from entering ANY list